services/app-api/forms/mcpar.json

```diff
@@ -224,20 +224,20 @@
           }
         },
         {
-          "name": "Add In Lieu of Services",
-          "path": "/mcpar/program-information/add-in-lieu-of-services",
+          "name": "Add In Lieu of Services and Settings",
+          "path": "/mcpar/program-information/add-in-lieu-of-services-and-settings",
           "pageType": "standard",
           "verbiage": {
             "intro": {
               "section": "Section A: Program Information",
-              "subsection": "In Lieu of Services and Settings Offered",
+              "subsection": "Add In Lieu of Services and Settings Offered",
               "info": [
                 {
                   "type": "p",
                   "children": [
                     {
                       "type": "html",
-                      "content": "If In Lieu of Services and Settings (ILOS) are available in this managed care program, enter the name of each ILOS offered, as approved in the managed care plan contract(s). Do not include short-term IMD stays. <b>You are required to complete this section if ILOS are available.</b> Do not include short-term IMD stays. Access "
+                      "content": "If In Lieu of Services and Settings (ILOS) are available in this managed care program, Enter the name of each ILOS, as approved in the managed care plan contract(s). <strong>Complete this section if ILOS are available in your program.</strong> Access "
```

Comment:
I saw `Enter` was capitalized after the comma in the spec, do we know why?

Reply:
probably just a typo, I'll update that! thanks